site/content/docs/main/backup-reference.md

```diff
@@ -19,3 +19,26 @@ To backup resources of specific Kind in a specific order, use option --ordered-r
 velero backup create backupName --include-cluster-resources=true --ordered-resources 'pods=ns1/pod1,ns1/pod2;persistentvolumes=pv4,pv8' --include-namespaces=ns1
 velero backup create backupName --ordered-resources 'statefulsets=ns1/sts1,ns1/sts0' --include-namespaces=ns1
 ```
+## Set Scheduled Backups
+
+The **schedule** operation allows you to back up your data at recurring intervals. These intervals are specified by a Cron expression.
+
+```
+velero schedule create NAME --schedule [flags]
+```
+
+For example, to create a backup that runs every day at 3 am run
+
+```
+velero create schedule backupName --schedule="0 0 3 ? * *"
```

Comment:
```suggestion
velero schedule create schedule-name --schedule="0 3 * * *"
```

1. the backupName is an invalid name to Kubernetes
2. I think the cronjob to run at 3 am every day is incorrect, ref to [link](https://crontab.guru/#0_3_*_*_*)

Reply:
thanks @jenting I updated these

Reply:
Thanks, the it should be "0 3 * * *"

Reply:
thank you for catching this @jenting, it should be fixed now.